### PR TITLE
Add Lombard Finality Provider

### PR DESCRIPTION
**Pull request by fruit37**
# New Lombard Finality Provider

## Checklist

- [x] I have followed the finality provider information registry
[guide](https://github.com/babylonlabs-io/networks/blob/main/bbn-1/finality-providers/README.md)
- [x] I have backed up my mnemonic
- [x] I have read and agree to the [Babylon Ecosystem Participant License](https://docs.babylonlabs.io/assets/files/babylon-ecosystem-participant-license.pdf) and the [Babylon Ecosystem Participant Agreement](https://docs.babylonlabs.io/assets/files/babylon-ecosystem-participant-agreement.pdf).

> [!CAUTION]
> The loss of the (generated keys + mnemonic) makes the finality provider
useless and unable to provide finality, which would lead to no transition to
later phases of the Babylon network.

### CODE REVIEW

**Comment by RafilxTenfen** [comment]
Hey @fruit37, can you update the PR description to follow this [PR template](https://github.com/babylonlabs-io/networks/blob/main/.github/PULL_REQUEST_TEMPLATE/new_finality_provider.md) and also check the checkboxes? 
Thanks!

**Comment by fruit37** [comment]
> Hey @fruit37, can you update the PR description to follow this [PR template](https://github.com/babylonlabs-io/networks/blob/main/.github/PULL_REQUEST_TEMPLATE/new_finality_provider.md) and also check the checkboxes? Thanks!

**Comment by RafilxTenfen** [comment]
@fruit37 should I reopen the PR?

**Comment by fruit37** [comment]
> @fruit37 should I reopen the PR?

@RafilxTenfen, yes, please. Thank you very much.

File: bbn-1/finality-providers/registry/Lombard.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": {
+    "moniker": "Lombard Finance",
```
Comment (RafilxTenfen) [inline bbn-1/finality-providers/registry/Lombard.json:3]: Can you make sure that the moniker does not contains white spaces
https://github.com/babylonlabs-io/networks/blob/main/bbn-1/finality-providers/README.md?plain=1#L228

After updating, you need to resign and update the `Lombard.sig`

Comment (RafilxTenfen) [reply bbn-1/finality-providers/registry/Lombard.json:3]: @fruit37

File: bbn-1/finality-providers/registry/Lombard.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": {
+    "moniker": "Lombard",
```
Comment (fruit37) [inline bbn-1/finality-providers/registry/Lombard.json:3]: @RafilxTenfen I have updated the files. Thanks.

File: bbn-1/finality-providers/registry/Lombard.json (hunk 1 above)
Comment (fruit37) [reply bbn-1/finality-providers/registry/Lombard.json]: just updated

**Review by RafilxTenfen — APPROVED** [review]
:100:

**Review by filippos47 — APPROVED** [review]

**Pull request merged**